### PR TITLE
DO NOT MERGE: Testing GitHub Review Functionality

### PR DESCRIPTION
**Pull request by CITguy**
**DO NOT MERGE!** Testing new Github Review functionality

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
 [![Build Status](https://travis-ci.org/rackerlabs/encore-ui.png?branch=master)](https://travis-ci.org/rackerlabs/encore-ui) [![Coverage Status](https://coveralls.io/repos/rackerlabs/encore-ui/badge.png?branch=master)](https://coveralls.io/r/rackerlabs/encore-ui?branch=master) [![devDependency Status](https://david-dm.org/rackerlabs/encore-ui/dev-status.png)](https://david-dm.org/rackerlabs/encore-ui#info=devDependencies)
```
Comment (CITguy) [inline README.md]: This is a comment for a review.

Comment (halkon) [inline README.md]: dsvfdsafsdf

Comment (halkon) [inline README.md]: sagsdgds

Comment (halkon) [inline README.md]: fsdads

Comment (KalpeshBhandari) [inline README.md]: @CITguy My first comment

Comment (KalpeshBhandari) [inline README.md]: Second comment

**Comment by CITguy** [comment]
## Findings

* User who submitted the PR will not be able to submit a review for changes.
* A user can submit multiple reviews requesting changes, but only the last review applies toward the PR checks.

### Recommended Review Workflow

1. Reviewer adds comments for review.
2. Reviewer finishes commenting with "Request Changes" option selected (opening a review check).
3. PR dev makes updates
4. Reviewer can add another review or approve their last review.

**Comment by CITguy** [comment]
reopening for additional testing

File: README.md (hunk 1 above)
Comment (KalpeshBhandari) [inline README.md]: Test 1 comment

File: README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (KalpeshBhandari) [inline README.md]: Second comment

**Pull request closed without merging**